tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Utils/Settings.cs

```diff
@@ -10,11 +10,30 @@ public sealed class Settings
 
         public string OrganisationApiBaseUrl { get; }
 
+        public SmtpServer Smtp { get; }
+
         public Settings(IConfigurationRoot config)
         {
             AdminConnectionString = config.GetConnectionString("CatalogueUsersAdmin");
             ConnectionString = config.GetConnectionString("CatalogueUsers");
             OrganisationApiBaseUrl = config.GetValue<string>("OrganisationApiBaseUrl");
+            Smtp = new SmtpServer(config.GetSection("SmtpServer"));
+        }
+
+        public sealed class SmtpServer
```

Comment:
Should probably be named SmtpServerSettings (as it's not an actual SmtpServer)

Reply:
👍 Good shout